src/common/k8s-api/endpoints/crd.api.ts

```diff
@@ -99,7 +101,11 @@ export class CustomResourceDefinition extends KubeObject {
   }
 
   getResourceUrl() {
-    return crdResourcesURL({
+    const di = getLegacyGlobalDiForExtensionApi();
+
+    const customResourcesRoute = di.inject(customResourcesRouteInjectable);
```

Comment:
Why not inject the `navigateToCustomResourcesInjectable` instead?

Reply:
Too big change for time being.

Reply:
okay, bit confused but okay.

Reply:
`navigateToCustomResourcesInjectable` provides function which navigates. This place requires URL, or we need to change all use places. Latter is bigger task which we scoped out based on checking use places.